dashboard/app/views/pd/workshop_mailer/_iste.html.haml

```diff
@@ -6,8 +6,8 @@
   practice.
 
 %p
-  We’re excited to offer ISTE membership exclusively to Code.org teachers at the special
-  price of $99 -- includes a special gift resource (value $37.95)! ISTE provides virtual
-  on-demand professional learning, in-person events and a vibrant online community of
-  educators around the world.
+  We’re excited to offer ISTE membership and a gift exclusively to Code.org teachers at
+  the special price of $99 -- includes a special gift resource (value $37.95)! ISTE
```

Comment:
You can remove this part `-- includes a special gift resource (value $37.95)` and change the sentence to end in a period.